NuKeeper/NuGet/Process/NugetPath.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+
+namespace NuKeeper.NuGet.Process
+{
+    public static class NugetPath
+    {
+        public static string Find()
+        {
+            var profile = Environment.GetEnvironmentVariable("userprofile");
+
+            if (string.IsNullOrWhiteSpace(profile))
+            {
+                throw new Exception("Could not find user profile path");
+            }
+
+            var commandlinePackageDir = Path.Combine(profile, ".nuget\\packages\\nuget.commandline");
```

Comment:
So obviously this is a pretty hard assumption/dependency.  Is it possible to include the nuget command line as a nuget package and refer to the one within the project?

I'm aware the dependency has existed for a little while but still